skyportal/tests/api/test_observing_runs.py

```diff
@@ -0,0 +1,133 @@
+from skyportal.tests import api
+
+
```

Comment:
Suggest adding a test for sunrise, sunset, and moon position/phase.

Reply:
See comment on other ephemeral data above.